liquibase/changes/feat_327.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE component
+  RENAME TO feature;
+DROP TABLE componentgtfsdataset;
+
```

Comment:
Suggestion for easier reading:
```suggestion
ALTER TABLE Component
  RENAME TO Feature;
DROP TABLE ComponentGtfsDataset;

```